src/Trinity.C/src/Network/ProtocolConstants.h

```diff
@@ -22,7 +22,7 @@ namespace Trinity
             PRESERVED_ASYNC,
         };
 
-        enum UInt32_Contants :uint32_t
+        enum UInt32_Constants :uint32_t
```

Comment:
could you please revert the changes in `Trinity.C/src/Network`? This part is heavily updated, here: https://github.com/Microsoft/GraphEngine/pull/227